homeassistant/components/nest/diagnostics.py

```diff
@@ -42,12 +45,18 @@ async def async_get_config_entry_diagnostics(
         return {"error": str(err)}
     if not nest_devices:
         return {}
-    return {
+    data: dict[str, Any] = {
         **diagnostics.get_diagnostics(),
         "devices": [
             nest_device.get_diagnostics() for nest_device in nest_devices.values()
         ],
     }
+    camera_data = await camera_diagnostics.async_get_platform_diagnostics(
```

Comment:
An alternative could be to make some helper for this and pass in PLATFORMS from __init__.py to try to load diagnostics for all platforms.

Reply:
let's not make it more complicated than necessary yet. We can always do that when we see more entity integrations require this.